ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -7,4 +7,4 @@ ref:
       cpu: 1000m
       memory: 2Gi
   documentation: |-
-    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform.
+    The IPI install step runs the OpenShift Installer in order to bring up an OpenShift cluster, using the provided cluster profile to choose a target IaaS platform. Anything that needs to be configured using manifests should generate them before this step and put them in the SHARED_DIR with the filename manifest_(manifest_name).yml so that this step can pull in the manifest in. 
```

Comment:
@AlexNPavel can we embed HTML here or something? I would love to see a fully-fledged example.

Reply:
we could allow markdown and use https://godoc.org/github.com/gomarkdown/markdown/html to convert to html for `steps.svc.`